internal/provider/resource_apply_module.go

```diff
@@ -191,6 +198,47 @@ func (t *applyModuleResource) Schema(_ context.Context, _ resource.SchemaRequest
 					},
 				},
 			},
+			"speculative": schema.BoolAttribute{
```

Comment:
I'm trying to make sense of this setting in terms of terraform. Could someone please elaborate on why someone would even set this to true since nothing is really created? How does a destroy work? A no-op?

Reply:
The idea here is that a user would be able to create speculative plans in several workspaces using a single orchestration workspace. The destroy would also create a speculative destroy plan.

Reply:
Is there something that needs to be done here?  Or, may I mark this comment/thread as resolved?

Reply:
After thinking about this some more, lets remove the `speculative` option because I think it's going to add confusion.